code/modules/sound/playsound.dm

```diff
@@ -0,0 +1,122 @@
+/**
+ * Sets up a playsound datum with the minimum required information to play the sound.
+ */
+/proc/playsound(source, sound)
```

Comment:
Probably better to rename it to something like create_sound/prepare_sound to avoid confusion from `playsound().play()`